src/pages/instances/forms/SecurityPoliciesForm.tsx

```diff
@@ -50,113 +55,192 @@ const SecurityPoliciesForm: FC<Props> = ({ formik, children }) => {
+              labelClassName={classnames({
+                "is-disabled": isContainerOnlyDisabled,
+              })}
+            />
+          ),
+        }),
+
+        getConfigurationRow({
+          formik: formik,
+          label: "Idmap size (Containers only)",
+          name: "security_idmap_size",
+          defaultValue: "",
+          disabled: isContainerOnlyDisabled,
+          children: (
+            <Input
+              id="security_idmap_size"
+              placeholder="Enter number"
+              name="security_idmap_size"
+              type="number"
+              min={0}
+              value={formik.values.security_idmap_size}
+              onBlur={formik.handleBlur}
+              onChange={formik.handleChange}
+              disabled={isContainerOnlyDisabled}
+              labelClassName={classnames({
+                "is-disabled": isContainerOnlyDisabled,
+              })}
+            />
+          ),
+        }),
+
+        getConfigurationRow({
+          formik: formik,
+          label: "Unique idmap usage (Containers only)",
+          name: "security_idmap_isolated",
+          defaultValue: "true",
+          disabled: isContainerOnlyDisabled,
+          children: (
+            <Select
+              id="security_idmap_isolated"
+              name="security_idmap_isolated"
+              onBlur={formik.handleBlur}
+              onChange={formik.handleChange}
+              options={booleanFieldsYesNo}
+              value={formik.values.security_idmap_isolated}
+              disabled={isContainerOnlyDisabled}
+            />
+          ),
+        }),
+
+        getConfigurationRow({
+          formik: formik,
+          label: "Allow /dev/lxd in the instance (Containers only)",
+          name: "security_devlxd",
+          defaultValue: "true",
+          disabled: isContainerOnlyDisabled,
+          children: (
+            <Select
+              id="security_devlxd"
+              name="security_devlxd"
+              onBlur={formik.handleBlur}
+              onChange={formik.handleChange}
+              options={booleanFieldsYesNo}
+              value={formik.values.security_devlxd}
+              disabled={isContainerOnlyDisabled}
+            />
+          ),
+        }),
+
+        getConfigurationRow({
+          formik: formik,
+          label:
+            "Make /1.0/images API available over /dev/lxd (Containers only)",
+          name: "security_devlxd_images",
+          defaultValue: "true",
+          disabled: isContainerOnlyDisabled,
+          children: (
+            <Select
+              id="security_devlxd_images"
+              name="security_devlxd_images"
+              onBlur={formik.handleBlur}
+              onChange={formik.handleChange}
+              options={booleanFieldsYesNo}
+              value={formik.values.security_devlxd_images}
+              disabled={
+                !formik.values.security_devlxd || isContainerOnlyDisabled
```

Comment:
The 2nd one used to be disabled, when the 1st on is not a yes. Seems like that change got lost in the middle of other changes.

Reply:
I relaxed the condition here - it gets rather complicated, because we'd have to figure out also inherited values eventually to make the right choice here. I think it is fine to treat both values independently to keep it simple.